lib/OA/Creative/File.php

```diff
@@ -183,7 +182,7 @@ function &factoryString($fileName, $content)
         }
 
         // Get new instance
-        $oCreative = &OA_Creative_File::factory($filePath, $fileName);
+        $oCreative = $this->factory($filePath, $fileName);
```

Comment:
Factory methods are normally static.

Reply:
But they are not in the **OA_Creative_File** Class. Do you want that I change the factory method instead?